src/renderer/controllers/torrent-list-controller.js

```diff
@@ -128,6 +128,26 @@ module.exports = class TorrentListController {
     }
   }
 
+  pauseAllTorrents () {
+    this.state.saved.torrents.forEach((torrentSummary) => {
+      if (torrentSummary.status === 'downloading') {
```

Comment:
I noticed that if a torrent is seeding it is not paused... Does it make sense to paused torrents that are seeding too?